app/controllers/api/v1/owners_controller.rb

```diff
@@ -37,14 +37,10 @@ def destroy
 
   def gems
     user = User.find_by_slug!(params[:handle])
-    if user
```

Comment:
As you've pointed out already, we should keep consistency and keep the plain error but change the `find_by` to not raise an exception.

Reply:
fixed.
Added `User.find_by_slug`, adding their unit tests, then going back to the fallback using non-exceptionable method.